conform-pr/main.go

```diff
@@ -17,9 +18,34 @@ func main() {
 	action := githubactions.New()
 	internal.DebugEnv(action)
 
+	errors := runChecks(action)
+
+	if len(errors) == 0 {
+		return
+	}
+
+	var serrors []string
+	for _, err := range errors {
+		serrors = append(serrors, fmt.Sprintf("%s", err))
+	}
+
+	action.Fatalf("The PR does not conform to the rules:\n - %s.", strings.Join(serrors, ";\n - "))
```

Comment:
I expect this part to be modified when we implement all the pretty markdown things.